symfony/framework-bundle/4.2/src/bootstrap.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use Symfony\Component\Dotenv\Dotenv;
+
+if (class_exists(Dotenv::class)) {
+    (new Dotenv())->loadForEnv($_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? 'test', __DIR__.'/../.env');
```

Comment:
Why not `dev` as a fallback if both `$_SERVER['APP_ENV']` and `$_ENV['APP_ENV']` are not set?

Reply:
You caught a bug! Thanks. Fixed in the last commit.